internal/common/agent_util_unix.go

```diff
@@ -1,8 +1,7 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: MIT
 
-//go:build unix
-// +build unix
+//go:build unix || linux
```

Comment:
Weird edge case where if there is a //go:build unix not allowing a call from linux env when the original file sanity_test.go does not contain a unix build tag but call a *_unix.go file.

Reply:
We should just change it to !windows.